scripts/build/TestImpactAnalysis/test_impact/base_test_impact.py

```diff
@@ -131,21 +139,38 @@ def __init__(self, args: dict):
         self._report_file = PurePath(self._report_workspace).joinpath(
             f"report.{self._instance_id}.json")
         args[ARG_REPORT] = self._report_file
-        self._parse_arguments_to_runtime(
-            args, self._runtime_args)
+        self._parse_arguments_to_runtime(args)
+
+    def _compile_multi_value_arg_string(self, multi_values):
+        """
+        Takes the multi values of a command line argument and returns a string with each value separated by a dash character.
+
+        @param multi_args: The multi values of the argument to compile as a dash-separated string.
+        """
 
-    def _parse_arguments_to_runtime(self, args, runtime_args):
+        num_values = len(multi_values)
+        multi_values_string = ""
+        for i in range(num_values):
+            multi_values_string += multi_values[i]
+            if i < num_values - 1:
+                multi_values_string += "-"
+
```

Comment:
You could probably replace this loop with
```suggestion
        multi_values_string = "-".join(multi_values)
```

Reply:
Fixed 👍